.github/workflows/pr.yaml

```diff
@@ -39,6 +41,18 @@ jobs:
     uses: rapidsai/shared-workflows/.github/workflows/conda-cpp-tests.yaml@branch-24.10
     with:
       build_type: pull-request
+  conda-java-build:
+    needs: checks
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/conda-java-build.yaml@branch-24.10
+    with:
+      build_type: pull-request
+  conda-java-tests:
+    needs: conda-java-build
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/conda-java-tests.yaml@branch-24.10
+    with:
+      build_type: pull-request
```

Comment:
There are [no conda-java-build or conda-java-tests workflows](https://github.com/rapidsai/shared-workflows/tree/branch-24.10/.github/workflows). We use a single "custom job" workflow for Java builds/tests in cuDF. Copy from here: https://github.com/rapidsai/cudf/blob/afd3a4b4776adf738284c9f0b99e1fc2fcefeec8/.github/workflows/pr.yaml#L153-L163

Reply:
Moved to use custom job